src/main/java/View/Support/ServletAbstracts/SessionServlet.java

```diff
@@ -14,7 +14,6 @@ public boolean checkPrivilege(HttpServletRequest req, HttpServletResponse resp)
         // Request has no session
         if (req.getSession(false) == null) {
             CustomLogs.Error("User trying to access site. [No session] FAILED.");
-            resp.sendError(HttpServletResponse.SC_UNAUTHORIZED);
```

Comment:
toto sa nahradilo metodou SendUnauthorized kvoli tomu ze to potom vie hodit exception?